web-gateway/app/controllers/Main.java

```diff
@@ -38,36 +38,4 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
         );
     }
 
```

Comment:
@TimMoore I removed formFactory and showInlineInstruction but userService need to be there even for index page request.